unyt/dimensions.py

```diff
@@ -152,6 +156,19 @@
 #: inductance
 inductance = inductance_mks = magnetic_flux_mks / current_mks
 
+# Cost basis units
+#: cost per length
+cost_per_length = cost_per_distance = currency / length
+#: cost per area
+cost_per_area = currency / area
+#: cost per volume
+cost_per_volume = currency / volume
+#: cost per energy
+cost_per_energy = cost_per_erg = currency / energy
+#: cost per power
+cost_per_power = currency / power
```

Comment:
I don't think these derived dimensions need to be added with explicit aliases in this namespace unless they're going to be used elsewhere inside `unyt`.